src/sentry/incidents/typings/metric_detector.py

```diff
@@ -49,24 +85,21 @@ def from_alert_rule_incident(
 
     @classmethod
     def from_workflow_engine_models(
-        cls, detector: Detector, issue_occurrence: IssueOccurrence
+        cls, detector: Detector, evidence_data: MetricIssueEvidenceData, group_status: GroupStatus
     ) -> AlertContext:
-        # TODO(iamrajjoshi): Finalize the fetch from issue_occurrence once we have evidence_data contract
-        threshold_type = issue_occurrence.evidence_data.get("threshold_type")
-        if threshold_type is not None:
-            threshold_type = AlertRuleThresholdType(threshold_type)
+        threshold_type = fetch_threshold_type(evidence_data)
+        resolve_threshold = fetch_resolve_threshold(evidence_data, group_status)
+        alert_threshold = fetch_alert_threshold(evidence_data, group_status)
+
         return cls(
             name=detector.name,
             action_identifier_id=detector.id,
             threshold_type=threshold_type,
             detection_type=detector.config.get("detection_type"),
             comparison_delta=detector.config.get("comparison_delta"),
-            # TODO(iamrajjoshi): Add sensitivity, alert_threshold, resolve_threshold
-            sensitivity=None,
-            resolve_threshold=None,
-            # Currently, i am hacking this so we don't have to fetch the alert_threshold, but we should
-            # remove this once we have the evidence_data contract
-            alert_threshold=1.0,
+            sensitivity=evidence_data.sensitivity,
```

Comment:
note from our IRL chat -- create a wrapper method for this to fetch from the data condition for now. then i can come through and update once the platform is adding this data.

(ps, pls add a #todo for @saponifi3d there so i can find it, since this would be an additive type change and not pop up when running mypy)